src/client/fuse_ll.cc

```diff
@@ -63,6 +65,18 @@
 
 #define _CEPH_CLIENT_ID	"ceph.client_id"
 
+/*
+ * The dedicated struct for snapid <-> stag map for each ceph
+ * inode, and the stag is a number in range [2, 0xffff], and
+ * the stag number 0 is reserved for CEPH_NOSNAP and 1 is
+ * reververd for CEPH_SNAPDIR.
```

Comment:
minor nit: s/reververd/reserved and What this stag actually used for?

Reply:
Fixed it. Thanks @neesingh-rh 

> What this stag actually used for?

The inode# will occupy 48 bits in ceph. And for none snapshot file/directory when you run the `stat myfile` the inode# is something like `100000000023`, and for snapshot files we will use the `((tag << 48) | inode#)` to fake the inode#, and in linux the faked inode# will be something like `10100000000023`.  The `tag` above is mapped from the `snapid`, because the `snapid` maybe larger than `0xFFFF` so we couldn't use the `snapid` to fake the inode# directly.